src/vs/workbench/contrib/editSessions/browser/editSessionsWorkbenchService.ts

```diff
@@ -173,30 +177,45 @@ export class EditSessionsWorkbenchService extends Disposable implements IEditSes
 			return true;
 		}
 
+		const authenticationSession = await this.getAuthenticationSession();
+		if (authenticationSession !== undefined) {
```

Comment:
*nit* could replace with just `if (!!authenticationSession)` here, above, and below